src/main/resources/database/schema_lockrepository_drop.sql

```diff
@@ -0,0 +1,11 @@
+BEGIN;
```

Comment:
According to the create .sql, this perhaps would require cleaning as well.

Reply:
cleaned up too :smile: